include/deal.II/base/parsed_convergence_table.h

```diff
@@ -0,0 +1,684 @@
+      {VectorTools::H1_norm, VectorTools::L2_norm, VectorTools::Linfty_norm}});
+
+  /**
+   * Full constructor for ParsedConvergenceTable.
+   *
+   * @param component_names Names of the components. Repeated consecutive
+   * names are interpreted as components of a vector valued field;
+   * @param list_of_error_norms Specify what error norms to compute for each
+   * unique component name;
+   * @param exponent The exponent to use in p-norms;
+   * @param extra_columns Extra columns to add. These may be "cells" or "dofs";
+   * @param rate_key Specify the extra column by which we will compute the
+   * error rates. This key can either be one of "cells" or "dofs", or, if you
+   * add extra columns to the table via the method add_extra_column(), it may
+   * be one of the extra columns you added;
+   * @param rate_mode Specify the rate mode to use when computing error rates.
+   * This maybe either "reduction_rate", "reduction_rate_log2", or "none". See
+   * the documentation of ConvergenceTable::RateMode for an explanation of how
+   * each of this mode behaves;
+   * @param error_file_name Name of error output file (with extension txt,
+   * gpl, tex, or org). If different from the empty string, than
+   * output_table() also writes in this file in the format deduced from its
+   * extension;
+   * @param precision How many digits to use when writing the error;
+   * @param compute_error Control wether the filling of the table is enabled
+   * or not. This flag may be used to disable at run time any error computation;
+   *
+   * The parameters you specify with this constructor can be written to a
+   * ParameterHandler object by calling the add_parameters() method. Once you
+   * call the add_parameters() method, the following options will be defined in
+   * the given ParameterHandler object, and the parameters of the instance of
+   * this class will follow the modification you make to the ParameterHandler
+   * object at run time:
+   * @code
+   * # Listing of Parameters
+   * # ---------------------
+   * # When set to false, no computations are performed.
+   * set Enable computation of the errors = true
+   *
+   * # Set this to a filename with extension .txt, .gpl, .org, or .tex to enable
+   * # writing the convergence table to a file.
+   * set Error file name                  =
+   *
+   * # Number of digits to use when printing the error.
+   * set Error precision                  = 3
+   *
+   * # Extra columns to add to the table. Available options are dofs and cells.
+   * set Extra columns                    = dofs, cells
+   *
+   * # The exponent to use when computing p-norms.
+   * set Exponent for p-norms             = 2
+   *
+   * # Each component is separated by a semicolon, and each norm by a comma.
+   * # Implemented norms are Linfty, L2, W1infty, H1, and custom. If you want to
+   * # skip a component, use none.
+   * set List of error norms to compute   = Linfty, L2, H1
```

Comment:
I don't think this example work anymore: we now parse `L2_norm`, not `L2`, since we use the `VectorTools` type. For the same reason I don't see how `custom` would now work.

If I have understood the custom column correctly we always supply a name for it anyway, so it shouldn't be here, right?

Reply:
You are right. I forgot to update this part of the documentation. This is now fixed.